### PR TITLE
Loading component

### PR DESCRIPTION
**Pull request by lorynmason**
closes issue #32 

changes:
- add loading component, styled and tested
- minor header css changes
- also added test coverage to package.json

note:
right now app is rendering Loading inside the switch statement, you can see it at "/loading", I'm not sure where/went we want to utilize the Loading component

### CODE REVIEW

**Comment by lorynmason** [comment]
I can't figure out why the build is failing, all the test are passing

File: package.json [hunk 1]
```diff
@@ -34,5 +34,11 @@
   "devDependencies": {
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.9.1"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx}",
+      "!src/index.js"
+    ]
```
Comment (mike-duke) [inline package.json]: going to end up with a minor conflict here: i did this also.

File: src/Components/App/App.js [hunk 2]
```diff
@@ -68,6 +70,7 @@ class App extends Component {
         <Header />
         <Message />
         <Switch>
+          <Route path="/loading" render={() => <Loading />}/>
```
Comment (mike-duke) [inline src/Components/App/App.js]: does this need to be a render?

File: src/Components/Loading/Loading.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Loading } from './Loading';
+import { shallow } from 'enzyme'
+
+describe('Loading', () => {
+  it('should match snapshot', () => {
+    let wrapper = shallow(<Loading />)
+    expect(wrapper).toMatchSnapshot()
+  });
+})
```
Comment (mike-duke) [inline src/Components/Loading/Loading.test.js]: thanks for setting this up!

File: src/styles/_Loading.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  }
+  50% { 
+    opacity: 0;
+    height: 45px;
+  }
+  100% { 
+    opacity: 1;
+    height: 50px;
+  }  
+}
```
Comment (mike-duke) [inline src/styles/_Loading.scss]: excited to see this!

**Pull request merged**